### PR TITLE
Fixed video previews not rendering after video-upload

### LINKED ISSUE #747: Video previews are not rendered
- [ ] Render the video player
- [ ] Improve the upload progress UI to remove the blank spaces.

https://vimeo.com/818467865/22365b1796

bug _t
@gaagul _a

Originally posted at: https://github.com/bigbinary/neeto-chat-web/issues/7156

### PR DESCRIPTION
**Pull request by gaagul**
- Fixes #747 

**Description**

- Fixed video previews not rendering after upload

**Checklist**

~~- [ ] I have made corresponding changes to the documentation.~~
~~- [ ] I have updated the types definition of modified exports.~~
- [x] I have verified the functionality in some of the neeto web-apps.
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).

**Reviewers**

@AbhayVAshokan

### CODE REVIEW

**Comment by gaagul** [comment]
@AbhayVAshokan _a Please review.

File: src/components/Editor/MediaUploader/LocalUploader.jsx [hunk 1]
```diff
@@ -89,7 +89,7 @@ const LocalUploader = ({
       }));
 
       setPendingUploads([]);
-      uploadedFiles.forEach(file => insertImageToEditor(file));
+      uploadedFiles.forEach(file => insertMediaToEditor(file));
```
Comment (AbhayVAshokan) [inline src/components/Editor/MediaUploader/LocalUploader.jsx:92]:
```suggestion
      uploadedFiles.forEach(insertMediaToEditor);
```
Shouldn't this work?

File: src/components/Editor/MediaUploader/index.jsx [hunk 2]
```diff
@@ -34,22 +34,36 @@ const MediaUploader = ({
...
+          .focus()
+          .setVideo({ src: url, caption, alt: filename })
+          .command(({ tr, commands }) => {
+            const { doc, selection } = tr;
+            const position = doc.resolve(selection.to).end() + 1;
+
+            return commands.insertContentAt(position, "<p></p>");
+          })
+          .run();
+  };
```
Comment (AbhayVAshokan) [inline src/components/Editor/MediaUploader/index.jsx:66]: A lot of logic is duplicated. It should be possible to clean it up.

```js
  const insertMediaToEditor = file => {
    const { url, filename = "", caption = "" } = file;
    const mediaAttrs = { src: url, caption, alt: filename };
    const setMedia = mediaUploader.image
      ? editor.chain().setFigure(mediaAttrs)
      : editor.chain().setVideo(mediaAttrs);

    setMedia
      .focus()
      .command(({ tr, commands }) => {
        const { doc, selection } = tr;
        const position = doc.resolve(selection.to).end() + 1;

        return commands.insertContentAt(position, "<p></p>");
      })
      .run();
  };
```

**Review by AbhayVAshokan — APPROVED** [review]
@gaagul _a please merge yourselves after making these changes and verifying the functionality.

**Pull request merged**